cypress/integration/test-api-key-required.js

```diff
@@ -14,7 +14,7 @@ describe(`Test API key required`, () => {
   })
 
   it('Should find a text in modal requesting API key', () => {
-    cy.contains('Enter your private API key')
+    cy.contains('Enter your Default Admin API key')
```

Comment:
```suggestion
    cy.contains('Enter your admin API key')
```

It does not need to be the default one